package.json

```diff
@@ -12,15 +12,15 @@
     "test:watch": "node node_modules/.bin/jest --watchAll --config config/jest.config.js",
     "test:current": "node node_modules/.bin/jest --watch --config config/jest.config.js",
     "coveralls": "cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
-    "storybook": "cross-env NODE_ENV=storybook start-storybook -p 6006",
-    "build-storybook": "cross-env NODE_ENV=storybook NODE_OPTIONS=--max_old_space_size=8192 build-storybook",
+    "stories": "tfm-stories",
+    "build-storybook": "tfm-build-stories",
     "deploy-storybook": "cross-env NODE_ENV=storybook NODE_OPTIONS=--max_old_space_size=8192 storybook-to-ghpages",
```

Comment:
`storybook-to-ghpages` doesn't work

Reply:
`deploy-storybook` should work AFAIK.

Since no one uses it today, I would strongly prefer not to fix it and come up with an opinionated way to do deployment next version. I got some cool ideas :)